x-pack/plugins/enterprise_search/public/applications/applications/components/search_application/search_preview/search_preview.tsx

```diff
@@ -407,7 +415,45 @@ export const SearchApplicationSearchPreview: React.FC = () => {
         <SearchProvider config={config}>
           <EuiFlexGroup>
             <EuiFlexItem>
-              <SearchBox inputView={InputView} />
+              <SearchBox
+                inputView={({ getInputProps }) => (
+                  <EuiFlexGroup gutterSize="s">
+                    <EuiFieldSearch
+                      fullWidth
+                      placeholder={i18n.translate(
+                        'xpack.enterpriseSearch.content.engine.searchPreview.inputView.placeholder',
```

Comment:
typo: the i18n keys should not contain engine

Reply:
I think, when I rebased messed up the translations. Reverted in 2989f2145e7ca252a5e2bcb27d0c2cdbc60352f8

There is a small typo in translations. Will create a separate PR. 

https://github.com/search?q=repo%3Aelastic%2Fkibana%20searchPreivew&type=code